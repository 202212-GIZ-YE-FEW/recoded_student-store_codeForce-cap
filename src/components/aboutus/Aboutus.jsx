import { withTranslation } from "next-i18next"

import Highlighter from "../highlighter"

import OurTeam from "../OurTeam"

function Aboutus({ t }) {
  return (
    <div
      className='flex flex-col items-center justify-center font-poppins'
      dir={t("language") === "ar" ? "rtl" : "ltr"}
    >
      <div className='grid  mt-20 items-center justify-center max-w-screen-md'>
        <Highlighter highlighterStyle='aboutus' text={t("mission")} />
        <p className='text-purple-dark lg:w-[700px] p-3 text-2xl leading-relaxed text-center'>
          {t("mission-des")}
        </p>
      </div>
      <div className='grid  mt-20 items-center justify-center max-w-screen-md'>
        <Highlighter highlighterStyle='aboutus' text={t("history")} />
        <p className='text-purple-dark lg:w-[700px] p-3 text-2xl leading-relaxed text-center'>
          {t("history-des")}
        </p>
      </div>
      <div className='grid  mt-20 items-center justify-center max-w-screen-md'>
        <Highlighter highlighterStyle='aboutus' text={t("goals")} />

        <p className='text-purple-dark lg:w-[700px] p-3 text-2xl leading-relaxed text-center'>
          {t("goals-des")}
        </p>
        <br />
      </div>
      <OurTeam />
    </div>
  )
}

export default withTranslation("aboutus")(Aboutus)
