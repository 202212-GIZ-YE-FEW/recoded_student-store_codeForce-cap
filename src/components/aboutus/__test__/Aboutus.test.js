import renderer from "react-test-renderer"

import Aboutus from ".."

it("renders correctly", () => {
  const tree = renderer.create(<Aboutus />).toJSON()
  expect(tree).toMatchSnapshot()
})
