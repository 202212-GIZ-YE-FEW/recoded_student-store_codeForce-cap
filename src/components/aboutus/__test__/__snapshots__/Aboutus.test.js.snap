// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex flex-col items-center justify-center font-poppins"
>
  <div
    className="flex flex-col mt-20 items-center justify-center max-w-screen-md"
  >
    <div
      className="flex flex-col justify-center items-center relative text-purple text-5xl font-bold"
    >
      <div
        className=" -rotate-3"
      >
        <img
          alt="highlighter icon"
          data-nimg="1"
          decoding="async"
          height={180}
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          src="/_next/image?url=%2Fimages%2FHighlighter.png&w=1200&q=75"
          srcSet="/_next/image?url=%2Fimages%2FHighlighter.png&w=640&q=75 1x, /_next/image?url=%2Fimages%2FHighlighter.png&w=1200&q=75 2x"
          style={
            {
              "color": "transparent",
            }
          }
          width={578}
        />
      </div>
      <div
        className="absolute mb-[57px] text-center"
      >
        <h2>
          Mission
        </h2>
      </div>
    </div>
    <p
      className="text-purple-dark w-[700px] p-3 text-2xl leading-relaxed text-center"
    >
      Our companys mission is to create a student-friendly marketplace that provides an affordable and convenient platform for students to buy, sell, and borrow various types of products. We aim to help students save money by providing them access to affordable used products while also helping them earn money by allowing them to sell their own items.
    </p>
  </div>
  <div
    className="flex flex-col mt-20 items-center justify-center max-w-screen-md"
  >
    <div
      className="flex flex-col justify-center items-center relative text-purple text-5xl font-bold"
    >
      <div
        className=" -rotate-3"
      >
        <img
          alt="highlighter icon"
          data-nimg="1"
          decoding="async"
          height={180}
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          src="/_next/image?url=%2Fimages%2FHighlighter.png&w=1200&q=75"
          srcSet="/_next/image?url=%2Fimages%2FHighlighter.png&w=640&q=75 1x, /_next/image?url=%2Fimages%2FHighlighter.png&w=1200&q=75 2x"
          style={
            {
              "color": "transparent",
            }
          }
          width={578}
        />
      </div>
      <div
        className="absolute mb-[57px] text-center"
      >
        <h2>
          History
        </h2>
      </div>
    </div>
    <p
      className="text-purple-dark w-[700px] p-3 text-2xl leading-relaxed text-center"
    >
      Our company was founded by a group of students who recognized the need for a platform that caters to the specific needs of students. They observed that students often struggle to find affordable used products or a reliable platform to sell their own items. The company was established with the idea of providing a safe and efficient platform for students to buy, sell, and borrow products.
    </p>
  </div>
  <div
    className="flex flex-col mt-20 items-center justify-center max-w-screen-md"
  >
    <div
      className="flex flex-col justify-center items-center relative text-purple text-5xl font-bold"
    >
      <div
        className=" -rotate-3"
      >
        <img
          alt="highlighter icon"
          data-nimg="1"
          decoding="async"
          height={180}
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          src="/_next/image?url=%2Fimages%2FHighlighter.png&w=1200&q=75"
          srcSet="/_next/image?url=%2Fimages%2FHighlighter.png&w=640&q=75 1x, /_next/image?url=%2Fimages%2FHighlighter.png&w=1200&q=75 2x"
          style={
            {
              "color": "transparent",
            }
          }
          width={578}
        />
      </div>
      <div
        className="absolute mb-[57px] text-center"
      >
        <h2>
          Goals
        </h2>
      </div>
    </div>
    <p
      className="text-purple-dark w-[700px] p-3 text-2xl leading-relaxed text-center"
    >
      Our primary goal is to become the go-to marketplace for students across the country. We aim to build a strong reputation as a trusted platform that offers a wide range of affordable and high-quality products. We also aim to continuously improve our platform to ensure that it remains easy to use, reliable, and secure for all users. Additionally, we strive to create a community of students who can share their experiences and insights about products and services, creating a more connected and supportive student network. Ultimately, we want to help students save money, earn money, and have a better experience as they navigate their academic journeys.
    </p>
  </div>
</div>
`;
